components/locale/bn_BD.tsx

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable no-template-curly-in-string */
+import Pagination from 'rc-pagination/lib/locale/bn_BD';
+import DatePicker from '../date-picker/locale/bn_BD';
+import TimePicker from '../time-picker/locale/bn_BD';
+import Calendar from '../calendar/locale/bn_BD';
+import { Locale } from '../locale-provider';
+
+const typeTemplate = '${label} টি সঠিক ${type} নয়।';
+
+const localeValues: Locale = {
+  locale: 'bn',
```

Comment:
![image](https://user-images.githubusercontent.com/6134547/124352889-d6cbf400-dc35-11eb-8c13-aa05cc2c577e.png)

Reply:
Which one is your right choice?

Reply:
`bn-bd` is correct. But, `bn` and `bn-bd` use the same strings and there is no difference. Should I language for both?

Reply:
No, just only switch to `bn-bd`.

Reply:
switched.